### PR TITLE
fakeroot: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/fakeroot.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# fakeroot
+
+> Run a command in an environment, faking root privileges for file manipulation.
+> More information: <https://manpages.debian.org/latest/fakeroot/fakeroot.1.html>.
```
Comment (pixelcmtd) [inline pages/linux/fakeroot.md:4]: Do you have any reason not to use manned.org?

Comment (marchersimon) [reply pages/linux/fakeroot.md:4]: Because `manned.org` fetches it from the Debian Manual.

File: pages/linux/fakeroot.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# fakeroot
+
+> Run a command in an environment, faking root privileges for file manipulation.
```
Comment (marchersimon) [inline pages/linux/fakeroot.md:3]:
```suggestion
> Run a command in an environment faking root privileges for file manipulation.
```
I think this makes more sense without a comma.

File: pages/linux/fakeroot.md (hunk 1 above)
Comment (navarroaxel) [reply pages/linux/fakeroot.md:4]: Yeah, this is a debian project <https://tracker.debian.org/pkg/fakeroot>.

**Review by CleanMachine1 — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
👌🏽

File: pages/linux/fakeroot.md (hunk 1 above)
Comment (pixelcmtd) [reply pages/linux/fakeroot.md:4]: Oh, I wasn't aware of that, only ever saw it on Arch.

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**